utils/editState.tsx

```diff
@@ -0,0 +1,51 @@
+// This file handles whether or not a user is in edit mode
+
+import React, { useContext, useState } from 'react';
+const LOCALSTORAGEKEY = 'tinaisadmin'
```

Comment:
Not a blocker, but we might want to better namespace this?  `tinacms.isEditing` or `tina.isEditing`?  I doubt we're going to brush up against existing keys and I don't know how much LOCALSTORAGE is used elsewhere.